src/lib.rs

```diff
@@ -225,9 +225,11 @@
     no_crate_inject,
     attr(deny(warnings/*, rust_2018_idioms*/), allow(dead_code, unused_variables))
 ))]
+#![feature(const_precise_live_drops)]
```

Comment:
Unfortunately, this prevents from building with stable rustc, which would not be acceptable.
Why is it required ? I see no code using it (at least in rustc 1.53, `bytes_from_borrowed` does not require it)